examples/outliers/alibi-detect-combiner/pipeline/outliersdetector/Detector.py

```diff
@@ -0,0 +1,36 @@
+import dill
+import json
+import os
+
+import numpy as np
+
+
+dirname = os.path.dirname(__file__)
+
+
+class Detector:
+    def __init__(self, *args, **kwargs):
+
+        with open(os.path.join(dirname, "preprocessor.dill"), "rb") as prep_f:
+            self.preprocessor = dill.load(prep_f)
+        with open(os.path.join(dirname, "model.dill"), "rb") as model_f:
+            self.od = dill.load(model_f)
+
+    def predict(self, X, feature_names=[]):
+        X_prep = self.preprocessor.transform(X)
+        od_preds = self.od.predict(X_prep, return_instance_score=True)
+        return json.dumps(od_preds, cls=NumpyEncoder)
```

Comment:
Same here, may be best to return dict so it uses jsonDict instead, although not critical as clearly it's working atm

Reply:
Will do.

Reply:
Done in https://github.com/SeldonIO/seldon-core/pull/1306/commits/a2839d4b63e09b138d18b549fef538583dcafec8.